### PR TITLE
Added custom fields (url, email, color etc) to API

### LINKED ISSUE #474: Create custom fields for email, urls and other data types
Create custom fields for url, email, color etc. This will help show better example responses and payloads in Swagger specs and possibly help us to validate those fields in a neat manner.

### PR DESCRIPTION
**Pull request by aviaryan**
Fixes #474 

* Better example data shown in Swagger UI
* Custom fields can be validated in POST/PUT request

### CODE REVIEW

File: open_event/api/custom_fields.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                value, str(self.__class__.__name__))
+            # raise MarshallingError
+            # disabling for development purposes as the server crashes when
+            # exception is raised. can be enabled when the project is mature
+        if self.__schema_type__ == 'string':
+            return unicode(value)
+        else:
+            return value
+
+    def validate(self, value):
```
Comment (juslee) [inline open_event/api/custom_fields.py]: Codacy Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2663229950/issues/source?bid=3233864&fileBranchId=3330981#l30)

Comment (juslee) [inline open_event/api/custom_fields.py]: Codacy Issue found: [Unused argument 'value'](https://www.codacy.com/app/triplez/open-event-orga-server/file/2663229950/issues/source?bid=3233864&fileBranchId=3330981#l30)

**Comment by aviaryan** [comment]
@juslee Please review. This is ready for merging.

**Comment by rafalkowalski** [comment]
@aviaryan  can i merge it?

**Comment by aviaryan** [comment]
will merge after resolving conflicts

**Pull request merged**